lib/httputils/json.go

```diff
@@ -0,0 +1,42 @@
+package httputils
+
+import (
+	"encoding/json"
+	"net/http"
+
+	common "boscoin.io/sebak/lib/common"
+)
+
+func WriteJSON(w http.ResponseWriter, code int, v interface{}) error {
+	w.Header().Set("content-type", "application/json")
+	w.WriteHeader(code)
+
+	if s, ok := v.(common.Serializable); ok {
```

Comment:
Doesn't `json.Marshal` do that itself ?

Reply:
It's a good question. :-)  Sometimes, `common.Serializable` is confusing for me. I need to think about it. And I have a question. The `common.Serializable` exists Only for DB Storage layer?

As you mentions, Just doing `json.Marhsal` make sense in API layer. Thanks :-)

Reply:
Oh wait, we have our own `Serializable` interface ? Why aren't we using `pkg/encoding`'s [`TextMarshaller`](https://golang.org/pkg/encoding/#TextMarshaler) which has the exact same definition and is recognized by the std lib ?

Reply:
First my thought,  `WriteJSON`  takes  `json.Marshaler` and handles it. But  From the result, I will change the `RenderSerializeFunc` to `RenderJSONFunc` too. ㅠㅠ  :-> 
I don't think the `TextMarshaller` is suitable for api rendering layer. IMHO.